dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/ResetPasswordTest.php

```diff
@@ -223,6 +223,25 @@ public function testPasswordResetForLockCustomer()
         $this->graphQlMutation($query);
     }
 
+    /**
+     * Check type of autocomplete_on_storefront storeConfig value
+     *
+     * @throws Exception
+     */
+    public function testReturnTypeAutocompleteOnStorefrontConfig()
+    {
+        $query = <<<QUERY
+{
+    storeConfig {
```

Comment:
Please set value using fixture and then perform test.

Consider moving to the different file since it is not related to the `ResetPassword`.

Reply:
Hi @lenaorobei 
I have used fixture for setting value also i moved this test to 
`dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/StoreConfigTest.php`
Thank you a lot for review.